go-controller/pkg/ovn/controller/services/load_balancer_test.go

```diff
@@ -434,13 +434,17 @@ func Test_buildServiceLBConfigs(t *testing.T) {
 							Protocol:   v1.ProtocolTCP,
 							TargetPort: outportstr,
 						}},
-						ExternalIPs: []string{"4.2.2.2", "42::42"},
```

Comment:
again same thing? we should probably add this back because if we have both set, it means we'd have the externalIP vip as well.